packages/lib/src/vis/shared/AxisSystem.module.css

```diff
@@ -1,21 +1,8 @@
-.axisSystem {
```

Comment:
This container no longer exists. The CSS Grid properties move to `.visCanvas` in `VisCanvas.module.css`. `pointer-events` and `z-index` move directly on `.axis` and `.grid`.